components/mail/postfix/postfix.p5m

```diff
@@ -26,6 +26,10 @@ set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
 license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
+<transform file -> add pkg.depend.bypass-generate libsasl\.so\.1>
+
+depend fmri=system/library/security/libsasl type=require
```

Comment:
why is it so complex?

Reply:
No clue why but pkg seems not to like libsasl. It can not identify libsasl as a dependency thus it needs to be bypassed during the depend recognition run and then added manually. This is how I saw other packages solved that particular issue.

Reply:
it seems pkgdepend finds dependency on libsasl fine here without this.